ITvO consultancy page (#515)
ubvu/open-handbook: topics/itvo-consultancy.qmd

---

> title: "ITVO Consultancy"

Reviewer: ```suggestion
title: "ITvO Consultancy"
```

categories: [Trainings and Knowledge Items]
# format: html
toc: true
# number-sections: false
---

> # ITvO: IT for Research

Reviewer: ```suggestion
## ITvO: IT for Research
```

> ITVO (IT voor Onderzoek) is a dedicated team within the university IT department, focused on supporting researchers with IT solutions. We bridge the gap between research needs and IT possibilities, making sure researchers have access to the right tools, infrastructure, and expertise.

Reviewer: ```suggestion
ITvO (IT voor Onderzoek) is a dedicated team within the university IT department, focused on supporting researchers with IT solutions. We bridge the gap between research needs and IT possibilities, making sure researchers have access to the right tools, infrastructure, and expertise.
```

> ## Our Services

Reviewer: ```suggestion
### Our Services
```

> - **SciStor**: Scientific storage solution of Vu Amsterdam. Data is stored on premise.

Reviewer: ```suggestion
- **SciStor**: Scientific storage solution of VU Amsterdam. Data is stored on premise.
```

- **SciCloud**: A flexible platform where researchers can create and manage virtual machines for diverse projects.
- **Ada HPC**: Access to High Performance Computing cluster of the Vu, for compute-intensive research tasks.

> - **Consultancy**: Personalized guidance to help researchers navigate IT options, optimize workflows, and solve technical challenges.

Reviewer: ```suggestion
- **Consultancy**: Personalised guidance to help researchers navigate IT options, optimise workflows, and solve technical challenges.
```

---

# ITVO Consultancy

Our consultancy service supports researchers at every stage of their project. We offer:

- **Needs Assessment**: Collaborating with you to understand your research goals and IT requirements.
- **Solution Matching**: Identifying the most suitable university and external IT resources for your project.
- **Technical Support**: Assisting with setup, optimization, and troubleshooting of research IT environments.
- **Best Practices**: Advising on data management, and security

Whether you are beginning a new project, expanding your work, or dealing with technical challenges, our team can help you find practical solutions and make the most out of the available IT resources.

> ## Contact

Reviewer: ```suggestion
### Contact
```
